about/about-network-policy.md

```diff
@@ -216,7 +216,7 @@ spec:
 
 #### Hierarchical policy
 
-[Calico Enterprise]({{site.baseurl}}/calico-enterprise/) supports hierarchical network policy using policy tiers. RBAC
+[Calico Enterprise](https://docs.tigera.io/master/security/tiered-policy) supports hierarchical network policy using policy tiers. RBAC
```

Comment:
Yikes, I don't think we should be linking to `/master` docs

Reply:
This was just placeholder...I was trying to get local build to review and went into the weeds trying to get it fixed. So then pushed to PR so I could review there. Yep, will fix it.

Reply:
Fixed.